CHANGES.rst

```diff
@@ -10,6 +10,8 @@ Changes
   subclass inheritance. See `PR 144
   <https://github.com/zopefoundation/zope.interface/pull/144>`_.
 
+- Add support for Python 3.8.
+
```

Comment:
I like two blank lines between ReST sections, but we haven't been consistent with this in the past.